testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/KeycloakConfiguration.java

```diff
@@ -184,13 +188,16 @@ private static ClientRepresentation createBearerOnlyClient(String clientId) {
         return client;
     }
 
-    private static UserRepresentation createUser(String username, String password, List<String> realmRoles) {
+    private static UserRepresentation createUser(String username, String password, List<String> realmRoles, String firstName, String lastName, Boolean emailVerified) {
```

Comment:
Because there are other usages of `createUser(username, password, realmRoles)`, this won't compile. We need to introduce a `createUser(String username, String password, List<String> realmRoles)` variant that calls this new method signature with the extra values set to null.

We should then update the set calls below so that the first name, last name, and email verified values are only set if not null.

Reply:
Thanks for pointing this out! 
Do you mean that there are other calls to `createUser` outside of this repo? I tried compiling just to `WildFly` project and that compiled fine locally. 
But if there are calls from other projects, then I can create a new function as you suggested.

Reply:
Oh sorry, I thought I saw some other calls to `createUser` in this file that hadn't been updated but I see now that these were updated to pass in `null` and `false`. So this will compile.

But to avoid needing to update the existing calls where the new values aren't needed, we can introduce a `createUser(String username, String password, List<String> realmRoles)` variant that calls your new variant with `null` and `false` for the two new params.